### ISSUE TITLE
Exception thrown when trying to use an invite-code without any more uses on it

### ISSUE THREAD

**Issue by pfraze** [ssbc/scuttlebot#297]
I used an invite code with 1 use twice. The second time crashed the pub.

```
/home/pfraze/scuttlebot/node_modules/secret-stack/node_modules/muxrpc/permissions.js:63
      opts.deny.forEach(function (path) {
                ^

TypeError: opts.deny.forEach is not a function
    at perms (/home/pfraze/scuttlebot/node_modules/secret-stack/node_modules/muxrpc/permissions.js:63:17)
    at module.exports (/home/pfraze/scuttlebot/node_modules/secret-stack/node_modules/muxrpc/permissions.js:71:12)
    at /home/pfraze/scuttlebot/node_modules/secret-stack/node_modules/muxrpc/index.js:57:15
    at setupRPC (/home/pfraze/scuttlebot/node_modules/secret-stack/index.js:105:67) 
    at /home/pfraze/scuttlebot/node_modules/secret-handshake/net.js:61:13
    at /home/pfraze/scuttlebot/node_modules/secret-handshake/secure.js:19:5
    at /home/pfraze/scuttlebot/node_modules/secret-handshake/handshake.js:90:11
    at /home/pfraze/scuttlebot/node_modules/secret-stack/index.js:91:21
    at /home/pfraze/scuttlebot/plugins/invite.js:44:18
    at /home/pfraze/scuttlebot/node_modules/secure-scuttlebutt/node_modules/level-sublevel/shell.js:102:12
```

**Comment by pfraze** [comment]
I've traced this to https://github.com/ssbc/scuttlebot/blob/master/plugins/invite.js#L99. It seems that muxrpc doesnt support `true` as the value for deny.

@dominictarr how should this be fixed?

**Comment by dominictarr** [comment]
fixed in 7.6.2

**Comment by dominictarr** [comment]
fixed in 7.6.2
